epplib/tests/tests_ietf/__init__.py

```diff
@@ -0,0 +1,4 @@
+# from epplib.tests.tests_ietf.constants import NAMESPACE, SCHEMA_LOCATION
```

Comment:
**question (non-blocking)**: We don't usually keep commented code in-place. It appears like this might be documentation for how to change the XML schemas. If so, could we include a comment explaining why this commented code is here in this file?

Reply:
Good catch! It took me a long time to work out the correct imports to be able to run _their_ test suite with their schema and _our_ tests with IETF schema. I forgot to clean this up.